packages/experimental/ExperimentalIcon/jest.config.js

```diff
@@ -0,0 +1,2 @@
+const { configureReactNativeJest } = require('@fluentui-react-native/scripts');
+module.exports = configureReactNativeJest('android');
```

Comment:
I think we should start not running all our jest tests on android by default - it might make sense to change this to win32. You'll need to add @office-iss/react-native-win32 to the package.json to get it working

Background - up until recently all our jest tests were running on Android, which meant that any files that were .win32.ts or .ios.ts or .macos.ts were not getting run. Being able to set the jest config to run on win32 and macOS is actually pretty recent (there were some issues that recently got fixed), so there's currently only one jest test where the jest config is set to win32

Reply:
Thank you Lynn, I didn't know that win32/mac support was done.

Reply:
Yep, it was pretty recent (<2 weeks ago). Please feel free to update the jest configs of any other components that would make more sense to run on win32! I think ideally in the future our jest tests are able to run on multiple platforms, but for now I'm hoping we can start by updating the jest tests to run on the platform that makes the most sense